QuickStart Fails on ES6 Features

REPRO STEPS
Follow the instructions at either the [Angular.io quickstart](https://angular.io/docs/ts/latest/quickstart.html), or the instructions on the [Angular2 GitHub QuickStart](https://github.com/angular/quickstart).

CURRENT BEHAVIOR
After running ``npm start``, a variety of error messages are raised, along the lines of:

``node_modules/@angular/common/src/facade/async.d.ts(27,33): error TS2304: Cannot find name 'Promise'.``

EXPECTED BEHAVIOR
The QuickStart should run by following the documented instructions.

--- a/README.md
+++ b/README.md
@@ -70,6 +70,10 @@ npm install
 npm start
 ```
 
+> If the `typings` folder doesn't show up after `npm install` please install them manually with:
+
+> `npm run typings -- install`
+
 The `npm start` command first compiles the application, 
 then simultaneously re-compiles and runs the `lite-server`.
 Both the compiler and the server watch for file changes.

